fe/fe-core/src/main/java/com/starrocks/load/loadv2/SparkLoadPendingTask.java

```diff
@@ -94,7 +95,7 @@ public class SparkLoadPendingTask extends LoadTask {
     public SparkLoadPendingTask(SparkLoadJob loadTaskCallback,
                                 Map<FileGroupAggKey, List<BrokerFileGroup>> aggKeyToBrokerFileGroups,
                                 SparkResource resource, BrokerDesc brokerDesc) {
-        super(loadTaskCallback, TaskType.PENDING);
+        super(loadTaskCallback, TaskType.PENDING, LoadPriority.NORMAL_VALUE);
```

Comment:
If we introduced the `QUEUEING` state, the `PENDING` state seems to have non-sense.

Reply:
i don't want to change already state, users may have sth depend on it.